students/Adil/Class_assignments/slicing.py

```diff
@@ -0,0 +1,34 @@
+# slicing assignment
+def first_last_exchange(seq):
+    return seq[-1:] + seq[1:len(seq)-1] + seq[:1]
+ 
```

Comment:
You can get the middle of the sequence minus the first and last by seq[1:-1]. The negative at the end means that drop that many from the end, not necessary to use len.